.github/workflows/django.yml.pass

```diff
@@ -0,0 +1,39 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+      working-directory: backend
+
+    - name: Create .env
+      run: cp .env.dist .env
+      working-directory: backend/backend
+
+    - name: Run Tests
+      run: |
+        python manage.py test
+      working-directory: backend
```

Comment:
Are the test run on PRs? I don't see any mention of them in the GitHub checks at the end of the PRs

Reply:
Are you talking about _this_ PR?
That's because the file is called `django.yml`**`.pass`**, I renamed it while tuning the two other actions